DotNet.DocsTools/GraphQLQueries/FilesInPullRequest.cs

```diff
@@ -74,14 +75,23 @@ public async IAsyncEnumerable<string> PerformQuery()
 
         var cursor = default(string);
         bool hasMore = true;
+        JsonElement jsonData = default;
         while (hasMore)
         {
             queryText.variables["cursor"] = cursor!;
-            var jsonData = await client.PostGraphQLRequestAsync(queryText);
-
+            try
+            {
+                jsonData = await client.PostGraphQLRequestAsync(queryText);
+            }
+            catch (InvalidOperationException) 
+            { 
+                hasMore = false;
+                break;
+            }
 
             var filesNode = jsonData.Descendent("repository", "pullRequest", "files");
-            (hasMore, cursor) = filesNode.NextPageInfo();
+                (hasMore, cursor) = filesNode.NextPageInfo();
```

Comment:
Is this indentation to align the variable names with the line above?

Reply:
No, that's a VS formatting issue. Updated.
```suggestion
            (hasMore, cursor) = filesNode.NextPageInfo();
```